.babelrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Because babel is configured in `rollup.config.js` (when bundling) and `karma.conf.js` (when testing), I think this file is not necessary and confusing.

Reply:
For me the build keeps returning an error from rollup until i added this file

Reply:
Do you use `npm run` command for invoking rollup?

`npm run build` command should invoke rollup (and other tools) with correct configs, like:

```console
$ npm run build

> c3@0.4.18 build /Users/kt3k/t/c3
> npm run build:js && npm run build:css


> c3@0.4.18 build:js /Users/kt3k/t/c3
> npm run build:js:rollup && npm run build:js:uglify


> c3@0.4.18 build:js:rollup /Users/kt3k/t/c3
> rollup -c > c3.js

(!) Some options have been renamed
https://gist.github.com/Rich-Harris/d472c50732dab03efeb37472b08a3f32
entry is now input
moduleName is now output.name
format is now output.format
...
```

`npm run` command automatically uses local rollup cli and you don't need the global one.

Reply:
yes, I do use this, but the issue is not rollup, rollup works but throws a different error (It looks like your Babel configuration specifies a module transformer), so I added this file and it worked for me, anyways I removed this file for you to merge.
when will you publish the next version to npm? we are still using our own library.